examples/simple/src/main.rs

```diff
@@ -121,12 +123,8 @@ impl ApplicationHandler for SimpleVelloApp<'_> {
                 // Re-add the objects to draw to the scene.
                 add_shapes_to_scene(&mut self.scene);
 
-                // Get the RenderSurface (surface + config)
-                let surface = &render_state.surface;
-
                 // Get the window size
-                let width = surface.config.width;
-                let height = surface.config.height;
+                let wgpu::SurfaceConfiguration { width, height, .. } = surface.config;
```

Comment:
I'd probably restore the explicit access to width and height here. I don't think this form is clearer, as `SurfaceConfiguration` is a weird type.